fastapi/routing.py

```diff
@@ -379,12 +383,14 @@ def __init__(
         self.responses = responses or {}
         response_fields = {}
         for additional_status_code, response in self.responses.items():
-            assert isinstance(response, dict), "An additional response must be a dict"
+            if not isinstance(response, dict):
+                raise AssertionError("An additional response must be a dict")
             model = response.get("model")
             if model:
```

Comment:
Reverse the condition and add the continue statement to decrease the indentation level.